packages/macro/package.json

```
@@ -29,6 +29,7 @@
   ],
   "dependencies": {
     "@babel/runtime": "^7.9.2",
+    "@lingui/conf": "^3.0.0-11",
```

Good catch! Just use placeholders for `@lingui/*` dependency versions, which is replaced with the correct version during build:

```suggestion
    "@lingui/conf": "0.0.0-managed-by-release-script",
```